app/helpers/solr_helper.rb

```diff
@@ -0,0 +1,10 @@
+# app/helpers/solr_helper.rb
+module SolrHelper
+  def solr_name(field_name, *opts)
+    if Module.const_defined?(:Solrizer)
```

Comment:
I am not user the dual boot support #rails_5_1? helper method here because that wouldn't necessarily be a valid check.